src/index.js

```diff
@@ -1,17 +0,0 @@
-import React from 'react';
-import ReactDOM from 'react-dom/client';
-import './index.css';
-import App from './App';
-import reportWebVitals from './reportWebVitals';
-
-const root = ReactDOM.createRoot(document.getElementById('root'));
-root.render(
-  <React.StrictMode>
-    <App />
-  </React.StrictMode>,
-);
-
-// If you want to start measuring performance in your app, pass a function
-// to log results (for example: reportWebVitals(console.log))
-// or send to an analytics endpoint. Learn more: https://bit.ly/CRA-vitals
-reportWebVitals();
```

Comment:
- [ ] Please do not delete this file because it is not boilerplate. If you delete this file, you would not be able to start your application. React expects the index.js to be present in the root directory. This is the error that I see when I try to start the application:

![image](https://user-images.githubusercontent.com/77106962/226686795-03baece4-c038-45cf-bd55-15fcd92665b9.png)

Hint: Kindly check the review summary up ⬆️ to see the files that need to be deleted apart from the ones you have deleted already 👍 😎

Reply:
As required, the following changes were implemented:
1 .Delete all the boilerplate from Create React App (text, images, styles).

![image](https://user-images.githubusercontent.com/108192399/226834316-f2accc71-9ca1-4f56-ba5b-6a4f8a920b8b.png)